src/core/Codend.Application/Projects/Commands/CreateProject/CreateProjectCommand.cs

```diff
@@ -58,18 +59,22 @@ public async Task<Result<Guid>> Handle(CreateProjectCommand request, Cancellatio
 
         var project = resultProject.Value;
 
-        var resultStatuses = DefaultTaskStatus.List
-            .Select(status => ProjectTaskStatus.Create(project.Id, status.Name))
+        // Create default statuses with positions.
+        var defaultStatusesList = DefaultTaskStatus.SortedList;
+        var statusesPositions = Lexorank.GetSpacedOutValuesBetween(defaultStatusesList.Count);
+        var i = 0;
+        var resultStatuses = defaultStatusesList
+            .Select(status => ProjectTaskStatus.Create(project.Id, status.Name, statusesPositions[i++]))
             .ToList();
         var result = resultStatuses.Merge();
         if (result.IsFailed)
         {
             return result.ToResult();
         }
 
-        // Set To-Do as first project default status.
-        var defaultStatus = resultStatuses.FirstOrDefault(status => 
-            status.Value.Name.Value == nameof(DefaultTaskStatus.ToDo))?.Value;
+        // Set To-Do as project default status.
+        var statuses = resultStatuses.Select(r => r.Value).ToList();
+        var defaultStatus = statuses.FirstOrDefault(status => status.Name.Value == DefaultTaskStatus.ToDo.Name);
```

Comment:
Maybe move it to extension method 💀

Reply:
nah, it is not going to be used anywhere else